eras/alonzo/test-suite/test/Tests.hs

```diff
@@ -29,7 +28,6 @@ defaultTests :: Spec
 defaultTests =
   describe "Alonzo tests" $ do
     AdaPreservation.tests @AlonzoEra 50
-    Golden.tests
```

Comment:
Did you intend to remove these tests?

I suspect what happened was you introduced a module with the same name into `cardano-ledger-alonzo:testlib:Test.Cardano.Ledger.Alonzo.Golden` as the one in `cardano-ledger-alonzo-test:library:Test.Cardano.Ledger.Alonzo.Golden` and this resulted in cabal complaining, which led to you removing this line.
Problem is that it effectively disables those tests, which we would like to keep, regardless how ugly they look.

An easy solution would be to simply moves the contents of this file: `eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/Golden.hs` intto 
`eras/alonzo/impl/test/Test/Cardano/Ledger/Alonzo/GoldenSpec.hs`.  Note that you'll also need to move the actual golden files used by that module.